packages/node_modules/@webex/internal-plugin-ediscovery/src/transforms.js

```diff
@@ -249,6 +249,54 @@ class Transforms {
             }));
         }
 
+        // Decrypt content url, web url and display name if extension is present
+        if(activity.extension){
+
+          //Decrypt content url for extension custom app
+          if(activity.extension.objectType =='extension' && activity.extension.extensionType =='customApp'){
```

Comment:
we could move this if condition above so that it could be reused by all the field population

Reply:
modified code for reusability.